internal/http/client.go

```diff
@@ -134,34 +143,6 @@ func (c *Client) Post(
 	return c.Do(req)
 }
 
-// RawPost behaves the same as Post, but without marshaling the body into JSON before making the request.
-// This is required at least in the case of Synthetics Labels, since the POST doesn't handle JSON.
-func (c *Client) RawPost(
```

Comment:
Collapsed this logic into `Post`, where if `body` is of type `[]byte` it is not marshaled. The `string` case here can be done inline with `Post(...., []byte(strBody), ...)`

Reply:
👏